install.sh

```diff
@@ -1122,7 +1269,7 @@ sandcats_register_name() {
   SANDCATS_CURL_PARAMS="${OVERRIDE_SANDCATS_CURL_PARAMS:-}"
 
   echo "Choose your desired Sandcats subdomain (alphanumeric, max 20 characters)."
-  echo "Type the word none to skip this step."
+  echo "Type the word none to skip this step, or help for help."
```

Comment:
I like not quoting this, fwiw, since otherwise people might type literally `"help"`.